fix(vscode): Do not use Microsoft marketplace for French extension

Jose-Matsuda opened the pull request:
-not supposed to use microsoft marketplace for cdr
-instead build from github repo
 -https://github.com/microsoft/vscode-loc/tree/release/1.50
-steps to build found in the wiki of said repo above
-removes downloaded files to keep image small

Example of run [here](https://streamable.com/vaklkg)

Jose-Matsuda commented:
**BEFORE REVIEW / MAKE CHANGES**
USE GIT CLONE TO GET THE REPO instead of **wget**'ing the zip file

**resolved**

wg102 commented on base/Dockerfile:
```diff
@@ -579,13 +578,21 @@ RUN \
-    echo "${SHA256fr} MS-CEINTL.vscode-language-pack-fr-${VS_FRENCH_VERSION}.vsix" | sha256sum -c - && \
-    bsdtar -xf MS-CEINTL.vscode-language-pack-fr-$VS_FRENCH_VERSION.vsix extension && \
-    rm MS-CEINTL.vscode-language-pack-fr-$VS_FRENCH_VERSION.vsix && \
+    #Install French Language extension via building and copying https://github.com/microsoft/vscode-loc/tree/release/1.50
```
- Why use version 1.50 and not the newer https://github.com/microsoft/vscode-loc/tree/release/1.52 ?
- Start the comment line with space

Jose-Matsuda replied:
My initial reasoning for using `1.50` as it's the second most up to date and I know that they would not mess with it. I do suppose that if it's under a **release** then they would not change it.

Jose-Matsuda replied:
Upon further review, I don't think `code-server` is compatible with 1.52 currently. The most recent version I can get to be recognized is `1.50.2
` 
![image](https://user-images.githubusercontent.com/23174198/102933464-4768ab80-4470-11eb-96d7-f35d31065621.png)

wg102 commented on base/Dockerfile:
```diff
@@ -579,13 +578,21 @@ RUN \
+    bsdtar -xf vscode-language-pack-fr-$VS_FRENCH_VERSION.vsix extension && \
     mv extension $HOME/.vscode/extensions/ms-ceintl.vscode-language-pack-fr-$VS_FRENCH_VERSION && \
+    cd ../../../ && \
+    # -fr as the clone becomes a write-protected regular file
```
Clearer comment would be useful

wg102 commented on base/Dockerfile:
```diff
@@ -579,13 +578,21 @@ RUN \
+    #Install French Language extension via building and copying https://github.com/microsoft/vscode-loc/tree/release/1.50
+    #https://github.com/Microsoft/vscode-loc/wiki/How-to-create-language-pack-extension#1-build-vsix 
+    VS_FRENCH_VERSION="1.50.2" && \
+    VS_LOCALE_REPO_VERSION="1.50" && \
```
See comment above about versions

Commit:
fix:vscode extension
-not supposed to use microsoft marketplace for cdr
-instead build from github repo
-https://github.com/microsoft/vscode-loc/tree/release/1.50
 -steps to build found in the wiki of said repo above
-removes downloaded files to keep image small
